{
  "submission_id": "S98",
  "user_id": "U5",
  "problem_id": "P1",
  "timestamp": "2025-10-09T11:40:42Z",
  "status": "Running... (Testcase 6)",
  "language": "python",
  "test_results": [
    {
      "test_case_number": 1,
      "status": "runtime_error",
      "message": "Runtime Error: Runtime Error (Exit Code: 1)",
      "execution_time": 0.01,
      "memory_usage": 7.88671875,
      "actual_output": "",
      "expected_output": "2",
      "input": "1 1\n"
    },
    {
      "test_case_number": 2,
      "status": "runtime_error",
      "message": "Runtime Error: Runtime Error (Exit Code: 1)",
      "execution_time": 0.05,
      "memory_usage": 7.8515625,
      "actual_output": "",
      "expected_output": "4",
      "input": "2 2\n"
    },
    {
      "test_case_number": 3,
      "status": "runtime_error",
      "message": "Runtime Error: Runtime Error (Exit Code: 1)",
      "execution_time": 0.05,
      "memory_usage": 7.953125,
      "actual_output": "",
      "expected_output": "0",
      "input": "0 0"
    },
    {
      "test_case_number": 4,
      "status": "runtime_error",
      "message": "Runtime Error: Runtime Error (Exit Code: 1)",
      "execution_time": 0.01,
      "memory_usage": 7.84765625,
      "actual_output": "",
      "expected_output": "12345",
      "input": "12345 0"
    },
    {
      "test_case_number": 5,
      "status": "runtime_error",
      "message": "Runtime Error: Runtime Error (Exit Code: 1)",
      "execution_time": 0.05,
      "memory_usage": 7.8671875,
      "actual_output": "",
      "expected_output": "67890",
      "input": "0 67890"
    },
    {
      "test_case_number": 6,
      "status": "runtime_error",
      "message": "Runtime Error: Runtime Error (Exit Code: 1)",
      "execution_time": 0.05,
      "memory_usage": 7.859375,
      "actual_output": "",
      "expected_output": "-30",
      "input": "-10 -20"
    }
  ]
}